confignode/src/main/java/org/apache/iotdb/confignode/client/AsyncDataNodeClientPool.java

```diff
@@ -58,13 +57,106 @@ public class AsyncDataNodeClientPool {
 
   private final IClientManager<TEndPoint, AsyncDataNodeInternalServiceClient> clientManager;
 
+  private final int retryNum = 6;
+
   private AsyncDataNodeClientPool() {
     clientManager =
         new IClientManager.Factory<TEndPoint, AsyncDataNodeInternalServiceClient>()
             .createClientManager(
                 new ConfigNodeClientPoolFactory.AsyncDataNodeInternalServiceClientPoolFactory());
   }
 
+  /**
+   * When the request fails, reconnect the DataNode that failed the request
+   *
+   * @param req request
+   * @param handlerMap Map<index, Handler>
+   * @param dataNodeLocations ConcurrentHashMap<index, TDataNodeLocation> The specific DataNodes
+   */
+  public void sendAsyncRequestToDataNodeWithRetry(
+      Object req,
```

Comment:
```suggestion
      Map<Integer, Object> reqMap
```
i.e. Map<DataNodeId, TReq>

Reply:
Except createregions, most requests should be a req, so Object is still used here